modules/cluster-logging-collector-log-forward-sts-cloudwatch.adoc

```diff
@@ -53,59 +55,102 @@ $ ccoctl aws create-iam-roles \
 --name=<name> \
 --region=<aws_region> \
 --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \
---identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com <1>
+--identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com #<1>
 ----
 <1> <name> is the name used to tag your cloud resources and should match the name used during your STS cluster install
-+
-. Apply the secret created:
+
+. Apply the secret by running the folloing command:
 [source,terminal]
 +
 ----
 $ oc apply -f output/manifests/openshift-logging-<your_role_name>-credentials.yaml
 ----
+
+. Create a credentials secret:
 +
+[source,yaml]
+----
+apiVersion: v1
+stringData:
+  credentials: |-
+    [default]
+    sts_regional_endpoints = regional
+    role_arn = <MY ROLE ARN>
```

Comment:
Note:   May be a point of confusion here, as logging doesn't use the secret exactly as the ccoctl expects, so most of this is unnecessary.   the arn is the only item we use from the secret and we will not parse anything else except the actual role_arn string (regardless of key).   We may consider matching this secret format with the other secret to avoid confusion.   For example, we do not parse the token path, so it is not necessary here.